ui/src/projects.vue

```diff
@@ -44,7 +44,8 @@
 
             <!--TODO - should refactor this.. similar to public y projects-->
             <div v-if="config.user" class="position: relative">
-                <h4 class="group-title">My Projects</h4>
+                <aupAgreementModal v-if="config.user && config.profile"></aupAgreementModal>                     
```

Comment:
how about only checking for `config.profile ?`

Reply:
I'd keep user as well, just to make sure heh